components/Card.js

```diff
@@ -11,10 +11,10 @@ export default function Card({ date, weight }) {
 
 const CardElement = styled.article`
   padding: 0 0.5em;
-  border: 1px grey solid;
-  background-color: lightgrey;
-  border-radius: 4%;
-  box-shadow: 5px 3px 1px grey;
+  border: 1px var(--text-primary) solid;
```

Comment:
I like that you work with CSS variables. 👍 I was confused about the order of your border property as I thought that the order is `width`, `style` and `color`, but this seems more of a convention as for [the property itself the order does not matter](https://developer.mozilla.org/en-US/docs/Web/CSS/border#syntax). Learned something new. :D

Reply:
Well that was actually an accident, I thought so too :D I changed this to prevent any other confusions in this code part.